test/powershell/Modules/Microsoft.PowerShell.Management/Alias.Tests.ps1

```diff
@@ -0,0 +1,51 @@
+Describe "Basic Validation of Alias Provider" -Tags "CI" {
+    BeforeAll {
+        $testAliasName = 'TestAlias'
+        $testAliasValue = 'Get-Date'
+    }
+
+    BeforeEach {
+        New-Item -Path Alias:\ -Name $testAliasName -Value $testAliasValue
+    }
+
+    AfterEach {
+        Remove-Item -Path "Alias:\${testAliasName}" -ErrorAction SilentlyContinue
+    }
+    
+    It "Test number of alias not Zero" {
+        $aliases = Get-ChildItem Alias:\
+        $aliases.Count | Should Not Be 0
+    }
+
+    It "Test alias 'dir'" {
+        $dirAlias = Get-Item Alias:\dir
+        $dirAlias.CommandType | Should Be 'Alias'
+        $dirAlias.Name | Should Be 'dir'
+        $dirAlias.Definition | Should Be 'Get-ChildItem'
+    }
+
+    It "Test creating new alias" {
+        $newAlias = New-Item -Path Alias:\ -Name 'NewTestAlias' -Value $testAliasValue
+        $newAlias.CommandType | Should Be 'Alias'
+        $newAlias.Name | Should Be 'NewTestAlias'
+        $newAlias.Definition | Should Be $testAliasValue
+        Remove-Item -Path 'Alias:\NewTestAlias' -ErrorAction SilentlyContinue
```

Comment:
Please move `Remove-Item` to a finally block, so that it will be cleaned up even if the above `Should Be` failed.